frontend/components/organisms/CourseDetailAccordionMenu.tsx

```diff
@@ -53,19 +77,24 @@ export function CourseDetailAccordionMenu({
       <Accordion allowToggle>
         {courseData.sections &&
           courseData.sections.map((section, sectionIndex) => {
+            const sectionProgress = getSectionCompletionPercentage(
+              sectionIndex,
+              session?.user?.id,
```

Comment:
```suggestion
              session.user.id,
```

上で
```  
    {session?.user?.id && (
```
の判定をしているので変更しても型のエラーは起きないはずです。